workspaces/github-actions/packages/app-next/src/components/auth/SignInPage.tsx

```diff
@@ -0,0 +1,37 @@
+import {
+  SignInPage as BackstageSignInPage,
+  SignInProviderConfig,
+} from '@backstage/core-components';
+import {
+  IdentityApi,
+  discoveryApiRef,
+  githubAuthApiRef,
+  useApi,
+  SignInPageProps,
+} from '@backstage/core-plugin-api';
+import React from 'react';
+import { setTokenCookie } from './cookieAuth';
+
+const githubProvider: SignInProviderConfig = {
+  id: 'github-auth-provider',
+  title: 'GitHub',
+  message: 'Sign In using GitHub',
+  apiRef: githubAuthApiRef,
+};
+
+export const SignInPage = (props: SignInPageProps) => {
+  const discoveryApi = useApi(discoveryApiRef);
+
+  return (
+    <BackstageSignInPage
+      {...props}
+      auto
+      providers={[githubProvider]}
+      onSignInSuccess={async (identityApi: IdentityApi) => {
```

Comment:
Don't think you need this cookie stuff, thats from the old auth which we want to avoid.

I think you should be able to get away with:

```ts
<BackstageSignInPage
  {...props}
  auto
  providers={[githubProvider]}
/>
```